themes/default/layouts/migrate/terraform.html

```diff
@@ -136,6 +139,24 @@ <h5>Coexist with Terraform state</h5>
             </li>
         </ul>
     </section>
-
-    {{ partial "hand-raise-section" . }}
+    <section id="migrate" class="container mx-auto w-full relative overflow-visible py-20">
```

Comment:
Adds a little padding so it isn't bleeding into the edges on mobile:

```suggestion
    <section id="migrate" class="container mx-auto w-full relative overflow-visible py-20 px-4">
```